packages/web/components/Landing/WhoAreWe.tsx

```diff
@@ -0,0 +1,245 @@
+import {
+  Box,
+  Container,
+  ListItem,
+  Text,
+  UnorderedList,
+  useBreakpointValue,
+} from '@metafam/ds';
+import BackgroundImageMobile from 'assets/landing/sections/section-5.sm.webp';
+import BackgroundImageDesktop from 'assets/landing/sections/section-5.webp';
+import { FullPageContainer } from 'components/Container';
+import { GuildFragment, Player } from 'graphql/autogen/types';
+import { Patron } from 'graphql/types';
+import { usePlayerFilter } from 'lib/hooks/player/players';
+import { useMotionDetector } from 'lib/hooks/useMotionDetector';
+import { useOnScreen } from 'lib/hooks/useOnScreen';
+import React, { useEffect, useRef, useState } from 'react';
+
+import { LandingNextButton } from './LandingNextButton';
+import { LandingPageSectionProps } from './landingSection';
+import { Rain } from './OnboardingGame/Rain';
+import { UserGrid } from './UserGrid';
+
+const elders = [
+  {
+    name: 'Griff Green',
+    img: 'https://pbs.twimg.com/profile_images/1639672195981926401/FecL3AYi_400x400.jpg',
+    link: 'https://twitter.com/thegrifft',
+  },
+  {
+    name: 'Zargham',
+    img: 'https://bafybeid364yqceeedrximgjcejam6plca2eqcdwi7hkmjjpvrap5au5xye.ipfs.dweb.link/',
+    link: '/player/mz',
+  },
+  {
+    name: 'James Young',
+    img: 'https://pbs.twimg.com/profile_images/1441126892623458304/dxwcOlSg_400x400.jpg',
+    link: 'https://twitter.com/jamesyoung',
+  },
+  {
+    name: 'Hanzi Freinacht',
+    img: 'https://pbs.twimg.com/profile_images/1339720086261800960/vdrivSWI_400x400.jpg',
+    link: 'https://twitter.com/HFreinacht',
+  },
+  {
+    name: 'Jeremy Akers',
+    img: 'https://pbs.twimg.com/profile_images/1661292382032207873/NRDsrDS1_400x400.jpg',
+    link: 'https://twitter.com/gospelofchange',
+  },
+  {
+    name: 'You?',
+    img: 'https://i.imgur.com/tdi3NGa.png',
+    link: '',
+  },
+  {
+    name: 'You?',
+    img: 'https://i.imgur.com/tdi3NGa.png',
+    link: '',
+  },
+];
+
+interface LandingPageSectionPropsWithPatronsGuildsAndElders
+  extends LandingPageSectionProps {
+  patrons: Patron[];
+  guilds: GuildFragment[];
+}
+
+export const WhoAreWe: React.FC<
+  LandingPageSectionPropsWithPatronsGuildsAndElders
+> = ({ section, nextSection, patrons, guilds }) => {
+  const ref = useRef<HTMLDivElement>(null);
+  const onScreen = useOnScreen(ref);
+  const root = typeof window !== 'undefined' ? document.body : null;
+  const noMotion = useMotionDetector(root);
+  const displayElement = noMotion ? true : !!onScreen;
+  const responsiveBg = useBreakpointValue({
+    base: BackgroundImageMobile,
+    md: BackgroundImageDesktop,
+  });
+
+  const { players } = usePlayerFilter();
+  const [topPlayers, setTopPlayers] = useState<Player[]>();
+
+  useEffect(() => {
```

Comment:
This state and effect hook could be replaced with a single `useMemo`

Reply:
swapped useEffect + useState with useMemo